homeassistant/components/renault/renault_entities.py

```diff
@@ -28,6 +27,7 @@ class RenaultDataEntityDescription(EntityDescription, RenaultDataRequiredKeysMix
 class RenaultEntity(Entity):
     """Implementation of a Renault entity with a data coordinator."""
 
+    _attr_has_entity_name = True
```

Comment:
It looks like it's also possible to set has_entity_name on the entity description. I don't have a lot of experience with entity descriptions yet, but when you have one is it preferred to use that instead? e.g. then everything name related is in one place

Reply:
There are different sub-classes of EntityDescription in use in Renault, and I want this to apply to ALL entities.
So in this particular case, I think it is easier to implement with an attribute.

IMO, class attributes are usually better than entity description, but entity description might be better than instance attributes.

Reply:
Thanks, makes sense to me.